src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,436 @@
+		$plugin = tribe( 'tec.main' );
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-admin-notice-install-event-tickets-js',
+			'admin/notice-install-event-tickets.js',
+			[
+				'jquery',
+				'tribe-common',
+			],
+			null,
+			[
+				'localize' => [
+					[
+						'name' => 'TribeEventsAdminNoticeInstall',
+						'data' => [ 'ajaxurl' => admin_url( 'admin-ajax.php', ( is_ssl() ? 'https' : 'http' ) ), ]
+					],
+				],
+				'groups'   => [
+					self::$assets_group,
+				],
+			]
+		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-admin-notice-install-event-tickets-css',
+			'admin/notice-install-event-tickets.css',
+			[ 'tec-variables-full' ],
+			[
+				'admin_enqueue_scripts',
+				'wp_enqueue_scripts',
+			],
+			[
+				'groups' => self::$assets_group,
+			]
+		);
+	}
+
+	/**
+	 * Get the plugin path for `Event Tickets`, by default.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return string $path The plugin path.
+	 */
+	protected function get_plugin_path( $slug = '' ): string {
+		if ( empty( $slug ) ) {
+			$slug = self::$plugin_slug;
+		}
+
+		return $slug . '/' . $slug . '.php';
+	}
+
+	/**
+	 * Checks if `Event Tickets` is installed.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return boolean True if active
+	 */
+	public function is_installed( $slug = '' ): bool {
+		$installed_plugins = get_plugins();
+
+		return array_key_exists( $this->get_plugin_path( $slug ), $installed_plugins ) || in_array( $this->get_plugin_path( $slug ), $installed_plugins, true );
+	}
+
+	/**
+	 * Checks if `Event Tickets` is active.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return boolean True if active.
+	 */
+	public function is_active( $slug = '' ): bool {
+		return is_plugin_active( $this->get_plugin_path( $slug ) );
+	}
+
+	/**
+	 * Check if we're on the classic "Install Plugin" page.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public function is_install_plugin_page(): bool {
+		return 'install-plugin' === tribe_get_request_var( 'action' );
+	}
+
+	/**
+	 * Should the `Install` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the install notice should be displayed.
+	 */
+	public function should_display_notice_install() {
+		return ! $this->is_installed()
+			&& empty( tribe_get_request_var( 'welcome-message-the-events-calendar' ) )
+			&& ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Should the `Activate` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the activate notice should be displayed.
+	 */
+	public function should_display_notice_activate() {
+		return $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Install notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_install() {
+		if ( ! $this->should_display_notice_install() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
```

Comment:
We don't need to call the assets directly, just let the conditional associated with both assets to load them in the proper order.